model/base_constraints.py

```diff
@@ -39,6 +40,7 @@ def __init__(
         self.add_maximum_demand_coverage(var.mu)
         self.add_deviation_from_ideal_demand(var.mu, var.delta)
         self.add_mapping_of_shift_to_demand(var.x, var.y)
+        #self.add_max_one_demand_cover_each_time(var.y)
```

Comment:
Is this intentional? If so, what is the purpose?

Reply:
The master branch did not have this constraint. Was wondering why it had not been added to the restrictions. Left it commented out until we had discussed this.

Reply:
Any thoughts @sondrenaerland ?

Reply:
It should be included though so might just be a miss that no one have seen.